app/mailers/subscription_mailer.rb

```diff
@@ -23,4 +23,11 @@ def notify_note_liked(node, user)
     @footer = feature('email-footer')
     mail(to: node.author.email, subject: subject).deliver
   end
+
+  def notify_tag_added(user)
```

Comment:
So, this is still in progress, pending the mailer view corresponding, here? Just checking, thanks!